.github/workflows/frontend_checks.yml

```diff
@@ -74,11 +74,8 @@ jobs:
       - name: Install dependencies
         run: npm install
 
-      - name: Build
-        run: npm run build
-
```

Comment:
`cy:start` already runs a build, so this was slowing down everything considerably building everything twice!

Reply:
Nice optimization